docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
     container_name: aerie_gateway
     depends_on: ["postgres"]
     environment:
-      AUTH_TYPE: none
+      AUTH_TYPE: cam
```

Comment:
Can we keep the default `none` here? Just to make the default config work if you are not on the JPL network.

Reply:
Oops, this was for my own testing. Reverting now.